docs/basic-features/data-fetching/incremental-static-regeneration.md

```diff
@@ -171,6 +171,29 @@ export async function getStaticProps() {
 }
 ```
 
+## Self-hosting ISR
+
+Incremental Static Regeneration (ISR) works on [self-hosted Next.js sites](/docs/deployment.md#self-hosting) out of the box when you use `next start`.
+
+You can use this approach when deploying to container orchestrators such as [Kubernetes](https://kubernetes.io/) or [HashiCorp Nomad](https://www.nomadproject.io/). By default, generated assets will be stored in-memory on each pod. This means that each pod will have its own copy of the static files. State data maybe shown until that specific pod is hit by a request.
+
+To ensure consistenly across all pods, you can disable in-memory caching. This will inform the Next.js server to only leverage storing assests generated by ISR in the file system.
```

Comment:
```suggestion
To ensure consistency across all pods, you can disable in-memory caching. This will inform the Next.js server to only leverage assets generated by ISR in the file system.
```